resources.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: default
+img: textbook.jpg
+img_link: http://algs4.cs.princeton.edu/home/
+caption: The course textbook
+title: resources
+active_tab: resources
+---
+
+<div>
+	<ul class="resource-header-list">
```

Comment:
@gindij 
It seems you're mixing tabs and spaces which makes for really bad indentation when viewed on Github or in a console editor etc. I don't use Sublime, but I think most sane editors allow you to set it to use one of tabs or spaces for it's auto-indentation. 

I have no personal preference over either (some people do), just don't mix the two. Go around all the files and fix any mixed tabs/spaces issues (easy way is to copy a tab or space character and then search for it in the file, the editor usually highlights the whitespace so you can fix any problems). There might also be plugins/tools that do this for you.